src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -162,7 +162,7 @@ m_cursorGrabbed   (false)
 
 
 ////////////////////////////////////////////////////////////
-WindowImplWin32::WindowImplWin32(VideoMode mode, const String& title, Uint32 style, const ContextSettings& /*settings*/) :
+WindowImplWin32::WindowImplWin32(const VideoMode& mode, const String& title, Uint32 style, const ContextSettings& /*settings*/) :
```

Comment:
I don't think that passing a bunch of integers by value is the slow thing in constructing a new window, let's keep it simple 😉

Reply:
Fix it.
https://github.com/SFML/SFML/pull/1732/commits/19b0c3dffc583de96cb3dfedfa82d4d2aa122725